.release/ci.hcl

```diff
@@ -129,21 +129,9 @@ event "post-publish-website" {
     on = "always"
   }
 }
-event "bump-version" {
-  depends = ["post-publish-website"]
-  action "bump-version" {
-    organization = "hashicorp"
-    repository = "crt-workflows-common"
-    workflow = "bump-version"
-  }
-
-  notification {
-    on = "fail"
-  }
-}
 
 event "update-ironbank" {
-  depends = ["bump-version"]
+  depends = ["post-publish-website"]
```

Comment:
@emilymianeil I just changed the dependency here, but I'm assuming the `update-ironbank` step should stick around. Is that correct?

Reply:
Yes thats correct!